src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -128,7 +128,8 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     <Reference Include="System.Security.Permissions" ExcludeAssets="All" />
     <Reference Include="System.Windows.Extensions" ExcludeAssets="All" />
 
-    <Reference Include="Microsoft.NETCore.App.Runtime.$(RuntimeIdentifier)"
+    <Reference Condition="'$(CrossgenOutput)' == 'true'"
```

Comment:
We don't want to make this reference conditional, some of the assemblies from here are necessary parts of our SharedFx. I'm thinking it'd be better to use another compatible RuntimeIdentifier in the event we're building on s390x

Reply:
@wtgodbe and I discussed this offline. Withdrawing the objection. Let's avoid the eng/Dependencies.props change and the "fix by adding a local repo holding the `Microsoft.NETCore.App.Runtime.linux-s390x` package to `NuGet.config`".